meeting_scheduler/meetingscheduler/src/main/java/com/spring/rest/meetingscheduler/exception/RoomCapacityInsufficientException.java

```diff
@@ -0,0 +1,15 @@
+package com.spring.rest.meetingscheduler.exception;
+
+public class RoomCapacityInsufficientException extends RuntimeException{
+    public RoomCapacityInsufficientException(String message) {
```

Comment:
remove different exceptions to a common exception class and vary them with mesaages instead of different files